style.css

```diff
@@ -1,3 +1,3 @@
 .iw_eventum {
-	background: transparent url(images/eventum.gif) 0px 1px no-repeat; padding-left: 16px;
+	background: transparent url(images/eventum.gif) 0px 0px no-repeat; padding-left: 20px;
```

Comment:
what's this change? attach screenshot of before and after of the css change please.